### PR TITLE
Renamed features

### PR DESCRIPTION
**Pull request by slinkydeveloper**
* Renamed `cloudevents-[lib]` features to `[lib]` naming
* Moved all the bindings code under `binding` module

Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -36,17 +36,17 @@
 //! Cloudevents uses a set of [feature flags] to conditionally compile
 //! only the module associated with the Protocol Binding you need:
 //!
-//! - `actix`: Enables the [actix] protocol binding module. This
+//! - `actix`: Enables the [`actix-web`](actix_web) protocol binding module. This
```
Comment (jcrossley3) [inline src/lib.rs:39]: I was thinking you'd link to `[binding::actix]` here. And then from there it has a link to `actix_web`.

File: src/lib.rs [hunk 2]
```diff
@@ -36,17 +36,17 @@
...
+//! [`event`](binding::actix::HttpResponseBuilderExt::event) function,
 //! and implementations for [`actix_web::FromRequest`] and
 //! [`actix_web::Responder`] in order to take advantage of actix-web's
 //! [Extractors] and [Responders]
-//! - `reqwest`: Enables the [reqwest] protocol binding module.
-//! - `warp`: Enables the [warp] protocol binding module.
-//! - `rdkafka`: Enables the [rdkafka] protocol binding module to
+//! - `reqwest`: Enables the [`reqwest`](reqwest_lib) protocol binding module.
+//! - `warp`: Enables the [`warp`](warp_lib) protocol binding module.
+//! - `rdkafka`: Enables the [`rdkafka`](rdkafka_lib) protocol binding module to
```
Comment (jcrossley3) [inline src/lib.rs:49]: Same here: link to the binding modules.

**Review by jcrossley3 — CHANGES_REQUESTED** [review]
Thanks for doing this!

**Comment by slinkydeveloper** [comment]
@jcrossley3 changed as suggested

**Review by jcrossley3 — APPROVED** [review]

**Pull request merged**